docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/cgov_home_landing.module

```diff
@@ -61,6 +61,10 @@ function cgov_home_landing_paragraph_access(EntityInterface $entity, $operation,
     return \Drupal::service('cgov_core.tools')->filterAccessForDependantEntity($entity, 'field_internal_link');
   }
 
+  if ($bundle === 'ncids_inline_video' && $operation === 'view') {
```

Comment:
Have you or @kate-mashkina tested this bit?

Reply:
yes, I tested it.  Note for @kate-mashkina , if you add the video, test using incognito once you add the unpublished video, otherwise you will have to log out of the CMS to see the change

Reply:
Awesome, thanks!